packages/storage/src/indexed-db/index.ts

```diff
@@ -23,6 +23,7 @@ import {
   AssetId,
   DenomMetadata,
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1alpha1/asset_pb';
+import { localExtStorage } from '../chrome';
```

Comment:
Instead of a direct import, it may be better to have storage or a function passed into the constructor

Reply:
This will also fix your test issue